langroid/agent/chat_agent.py

```diff
@@ -179,8 +187,16 @@ def __init__(
         self.output_format: Optional[type[ToolMessage | BaseModel]] = None
 
         self.saved_requests_and_tool_setings = self._requests_and_tool_settings()
+        # Tracks if we have enabled use of an output format
+        # We disable use when the output format is changed for any such tool
+        # where `enable_message` was not called explicitly
+        self.enabled_use_output_format: Optional[type[ToolMessage]] = None
+        # As above but used to disable handling
```

Comment:
Change comment to -- 

As above but deals with "enabled for handling" rather than "enabled for use".